### PR TITLE
Enable build and run commands with relative path.

### PR DESCRIPTION
**Pull request by parkavi11**
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/18786

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [x] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: tests/jballerina-integration-test/src/test/resources/packaging/singleBalFile/relative/testBal/hello_world.bal [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import ballerina/io;
```
Comment (kalaiyarasiganeshalingam) [inline tests/jballerina-integration-test/src/test/resources/packaging/singleBalFile/relative/testBal/hello_world.bal]: Missing license header

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java [hunk 2]
```diff
@@ -172,101 +178,113 @@ public void execute() {
             CommandUtil.exitError(this.exitWhenFinish);
             return;
         }
-    
+
+        if (this.nativeBinary) {
```
Comment (warunalakshitha) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java]: Do we need this?

Comment (warunalakshitha) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java]: Do we need this?

**Comment by warunalakshitha** [comment]
Also shall we check with --source-root as well.

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java (hunk 2 above)
Comment (parkavi11) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java]: Removed.

**Pull request closed without merging**